spec/controllers/download_controller_spec.rb

```diff
@@ -3,17 +3,20 @@
 RSpec.describe DownloadController, type: :controller do
   include_context "two flows"
   describe "GET index" do
+    before(:all) do
+      context = Context.first
```

Comment:
this is probably not necessary (same in the other controller spec) - context variable is defined in the included rspec context and so in this situational context (!) probably no need to redefine it ;) if it ends up being nil maybe there's some other problem